<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>188</width>
      <height>534</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Box</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="labelBoxZ">
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderZBox">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumberBoxZ">
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Sphere</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="labelSphereRadius">
             <property name="text">
              <string>r</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderSphereRadius">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumberSphereRadius">
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Ellipsoid</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="labelRX">
               <property name="text">
                <string>rx</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSliderRX">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumberRX">
               <property name="segmentStyle">
                <enum>QLCDNumber::Flat</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="labelRY">
               <property name="text">
                <string>ry</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSliderRY">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumberRY">
               <property name="segmentStyle">
                <enum>QLCDNumber::Flat</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="labelRZ">
               <property name="text">
                <string>rz</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSliderRZ">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumberRZ">
               <property name="segmentStyle">
                <enum>QLCDNumber::Flat</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="120,40">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="labelRed">
                 <property name="text">
                  <string>r</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderRed">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberRed">
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="labelGreen">
                 <property name="text">
                  <string>g</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderGreen">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberGreen">
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="labelBlue">
                 <property name="text">
                  <string>b</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderBlue">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberBlue">
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="labelAlpha">
                 <property name="text">
                  <string>a</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="horizontalSliderAlpha">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberAlpha">
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Color" name="widgetColor" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>551</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10" stretch="70,0">
     <item>
      <widget class="Plotter" name="widgetPlotter" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QSlider" name="horizontalSliderAxisZ">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumberAxisZ">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbreDim"/>
   <addaction name="actionAbreSaveOFF"/>
   <addaction name="actionPutVoxel"/>
   <addaction name="actionCutVoxel"/>
   <addaction name="actionPutBox"/>
   <addaction name="actionCutBox"/>
   <addaction name="actionPutSphere"/>
   <addaction name="actionCutSphere"/>
   <addaction name="actionPutEllipsoid"/>
   <addaction name="actionCutEllipsoid"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionAbreDim">
   <property name="text">
    <string>Grid Dim</string>
   </property>
   <property name="toolTip">
    <string>AbreDim</string>
   </property>
  </action>
  <action name="actionAbreSaveOFF">
   <property name="text">
    <string>SaveOFF</string>
   </property>
   <property name="toolTip">
    <string>abreSaveOFF</string>
   </property>
  </action>
  <action name="actionPutVoxel">
   <property name="text">
    <string>PutVoxel</string>
   </property>
  </action>
  <action name="actionCutVoxel">
   <property name="text">
    <string>CutVoxel</string>
   </property>
  </action>
  <action name="actionPutBox">
   <property name="text">
    <string>PutBox</string>
   </property>
  </action>
  <action name="actionCutBox">
   <property name="text">
    <string>CutBox</string>
   </property>
  </action>
  <action name="actionPutSphere">
   <property name="text">
    <string>PutSphere</string>
   </property>
  </action>
  <action name="actionCutSphere">
   <property name="text">
    <string>CutSphere</string>
   </property>
  </action>
  <action name="actionPutEllipsoid">
   <property name="text">
    <string>PutEllipsoid</string>
   </property>
  </action>
  <action name="actionCutEllipsoid">
   <property name="text">
    <string>CutEllipsoid</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Color</class>
   <extends>QWidget</extends>
   <header>color.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberAlpha</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberBlue</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberGreen</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRed</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAxisZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberAxisZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>769</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberBoxZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSphereRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberSphereRadius</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRZ</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
